src/plugins/OclPlugin/OclPrinter.cs

```diff
@@ -0,0 +1,476 @@
+﻿namespace OclPlugin
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Antlr4.Runtime.Misc;
+    using EditorPluginInterfaces;
+    using Repo;
+
+    internal class OclPrinter : OclBaseVisitor<bool>
+    {
+        private readonly ArrayList<Dictionary<string, object>> vars;
+        private readonly OclCalc calc;
```

Comment:
```suggestion
        private readonly OclCalculator calculator;
```
Буквы пока бесплатны, пользуйтесь этим.